cli/report/report.py

```diff
@@ -374,6 +383,22 @@ def add_passing_job_comment(self, job: Job, jira: Jira, issue_id: str) -> None:
                             """
         jira.comment(issue_id=issue_id, comment=comment)
 
+    def add_passing_job_label(self, jira: Jira, issue_id: str) -> None:
+        """
+        Used to add a label on a Jira issue that is open but has had a passing job since the issue was filed.
+
+        Args:
+            jira (Jira): Jira object.
+            issue_id (str): Issue ID of the open issue to comment on.
+
+        Returns:
```

Comment:
No need to specify `Returns` is None

Reply:
This was added to maintain the doctring style established by  [this commit](https://github.com/RedHatQE/firewatch/commit/ed545f60a03d6e9bc28cecbc79f2923c1575129e#diff-09fb611b2afe245a575054b1c0535365566456c8e5c8244120176901409e3631R233). Should the remaining docstrings that use this style be refactored separately as well?

Reply:
up to you, but there is no need to add `Returns:` if nothing is returned